addons/payment_stripe/models/payment.py

```diff
@@ -277,6 +277,9 @@ def _stripe_s2s_validate_tree(self, tree):
             self.write(vals)
             self._set_transaction_pending()
             return True
+        if status == 'requires_payment_method':
```

Comment:
elif

Reply:
I don't this is required here. `status` value is always different (one value per function call) and there are `return` in the `if` statements.